src/qt/modaloverlay.h

```diff
@@ -7,44 +7,51 @@
 
 #include <QDateTime>
 #include <QWidget>
+#include <memory>
 
 //! The required delta of headers to the estimated number of available headers until we show the IBD progress
-static constexpr int HEADER_HEIGHT_DELTA_SYNC = 24;
+constexpr int HEADER_HEIGHT_DELTA_SYNC = 24;
+#if defined(Q_OS_WIN)
+extern	std::weak_ptr<int> com_r;
+#endif 
 
 namespace Ui {
-    class ModalOverlay;
+	class ModalOverlay;
 }
 
 /** Modal overlay to display information about the chain-sync state */
 class ModalOverlay : public QWidget
 {
-    Q_OBJECT
+	Q_OBJECT
 
 public:
-    explicit ModalOverlay(QWidget *parent);
-    ~ModalOverlay();
+	explicit ModalOverlay(QWidget *parent);
+	~ModalOverlay();
 
 public Q_SLOTS:
-    void tipUpdate(int count, const QDateTime& blockDate, double nVerificationProgress);
-    void setKnownBestHeight(int count, const QDateTime& blockDate);
+	void tipUpdate(int count, const QDateTime& blockDate, double nVerificationProgress);
+	void setKnownBestHeight(int count, const QDateTime& blockDate);
 
-    void toggleVisibility();
-    // will show or hide the modal layer
-    void showHide(bool hide = false, bool userRequested = false);
-    void closeClicked();
-    bool isLayerVisible() const { return layerIsVisible; }
+	void toggleVisibility();
+	// will show or hide the modal layer
+	void showHide(bool hide = false, bool userRequested = false);
+	void closeClicked();
+	bool isLayerVisible() const { return layerIsVisible; }
 
 protected:
-    bool eventFilter(QObject * obj, QEvent * ev);
-    bool event(QEvent* ev);
+	bool eventFilter(QObject * obj, QEvent * ev);
+	bool event(QEvent* ev);
```

Comment:
it is unnecessary to replace the space indentation with tabs indentation, please don't.